### PR TITLE
Reorder prerequesites check

### LINKED ISSUE #297: Not a clear message in cli and web if owncloud is not installed
bildschirmfoto von 2016-04-06 21-18-13

Action items:
- verify if cloud is installed
- display message in console to the user if not

### PR DESCRIPTION
**Pull request by VicDeo**
Fixes #297  Not a clear message in cli and web if owncloud is not installed

Note: as soon as 
1. logging is done to datadir   (as required by @LukasReschke)
2. OC should be installed to find its datadir 
there is also a fallback logging to stdout 
notinstalled

### CODE REVIEW

**Comment by VicDeo** [comment]
@DeepDiver1975

**Comment by VicDeo** [comment]
@DeepDiver1975 please review

**Comment by VicDeo** [comment]
@DeepDiver1975 @PVince81 review required

File: src/Resources/views/partials/error.php [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?=$this->e($error) ?>
```
Comment (PVince81) [inline src/Resources/views/partials/error.php]: no header?

**Comment by PVince81** [comment]
Code looks fine :+1:

**Comment by PVince81** [comment]
@DeepDiver1975 please review and test as you are the bug reporter

**Comment by VicDeo** [comment]
@DeepDiver1975 Please have a look

**Comment by VicDeo** [comment]
Stable9.1: https://github.com/owncloud/updater/pull/394

**Pull request merged**